deployment_steps_cfn.adoc

```diff
@@ -1,9 +1,9 @@
-After clicking the *Deploy* link on the Quick Start landing page, you will be brought to the https://console.aws.amazon.com/cloudformation[CloudFormation console], with the template URL pre-populated. Ensure you are in the correct Region and click *Next* to enter parameter values.
+After you choose a deployment option, the https://console.aws.amazon.com/cloudformation[AWS CloudFormation console^] opens with a prepopulated template. Ensure that you are in the correct Region, and choose *Next* to enter parameter values.
 
-NOTE: Unless you are customizing the Quick Start templates for your own deployment projects, keep the default settings for the parameters *Quick Start S3 bucket name*, *Quick Start S3 bucket Region*, and *Quick Start S3 key prefix*. Changing these settings automatically updates code references to point to a new Quick Start location. For more information, see the https://aws-quickstart.github.io/option1.html[AWS Quick Start Contributor’s Guide^].
+NOTE: Unless you are customizing the Quick Start templates for your own projects, don't change the default settings for the parameters *Quick Start S3 bucket name*, *Quick Start S3 bucket Region*, and *Quick Start S3 key prefix*. Changing these settings automatically updates code references to point to a new Quick Start location. For more information, see the https://aws-quickstart.github.io/option1.html[AWS Quick Start Contributor’s Guide^].
 
```

Comment:
Looks like we're missing two steps:

. On the *Create stack* page, keep the default setting for the template URL, and then choose *Next*.

. On the *Specify stack details* page, change the stack name if needed. Review the parameters for the template. Provide values for the parameters that require input. For all other parameters, review the default settings and customize them as necessary.